draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -159,14 +159,31 @@ The ICE agent SHOULD, if available, use the first IPv6 address resolved, otherwi
 
 A peer-reflexive remote candidate could be learned and constructed from the
 source transport address of the STUN Binding request as an ICE connectivity
-check. The peer-reflexive candidate could share the same address as a remote mDNS
-ICE candidate whose name is
-in the process of being resolved. In addition to the elimination procedure
+check. The peer-reflexive candidate could share the same address as a remote
+mDNS candidate that is in the process of being signaled or 
+name resolution. See the example below, in which the
+initiating web application learns the IP of the remote peer via a
+peer-reflexive candidate, despite using mDNS.
+
+            ICE Agent 1 (1.1.1.1)                     ICE Agent 2 (2.2.2.2)
+      <Register     |                                         |
+      mDNS name N1> |                                         |
+                    |----------- mDNS Candidate N1 ---------->|
+                    |                                         | <Resolve 
+                    |                                         | mDNS name N1>
+                    |<======== STUN check to 1.1.1.1 =========|
+    prflx candidate |                                         |
+    2.2.2.2 created |                                         | <Register
+                    |                                         | mDNS name N2>
+                    |<---------- mDNS Candidate N2 -----------|                                         
+                    |                                         |
+
+In addition to the elimination procedure
```

Comment:
```suggestion
ICE Agent 1 (1.1.1.1)                                       ICE Agent 2 (2.2.2.2)
      <Register           |                                         | <Register
      mDNS name N1>       |                                         | mDNS name N2>                                       
                          |----------- mDNS Candidate N1 ---------->|
                          |<---------- mDNS Candidate N2 -----------|        
<Resolve                  |                                         | <Resolve 
 ...                      |                                         | mDNS name N1>
 mDNS                     |<======== STUN check to 1.1.1.1 =========|
 ...     <prflx candidate |                                         |
 name    2.2.2.2 created> |                                         |
 ...                      |                                         |
 N2>                      |                                         |

In addition to the elimination procedure
```

Reply:
I added one more example for the case when prflx is created during the resolution. We may need to the edit the sentence before the example if we would like add this one to the text as well.

Reply:
There are probably more scenarios thinkable (since signaling is out of band from the STUN checks), but I think there is value in both examples.